docs/troubleshooting.md

```diff
@@ -34,7 +34,7 @@ redhat-operators      appregistry   https://quay.io/cnr   redhat-operators
 ```
 
 If the status of any of the OperatorSource is anything other than `Succeeded`, you can investigate the OperatorSource with 
-`oc describe opsrc <Name-of-OperatorSource> -n openshift-marketplace`. [This section](#operatorSource-failing-to-download-from-datastore) of the doc also contains helpful information about debugging an OperatorSource. 
+`oc describe opsrc <Name-of-OperatorSource> -n openshift-marketplace`. [This section](#operatorSource-failing-to-connect-to-the-datastore) and [the following section](#operator(s)-in-an-operatorsource-fail-to-show-up-in-the-ui) of the doc also contains helpful information about debugging an OperatorSource. 
```

Comment:
the `the following section` link doesn't work

Reply:
See https://github.com/operator-framework/operator-marketplace/blob/b1bf31056e8a6dec8c468d2fbc3e1477df65e1ee/docs/troubleshooting.md#operator(s)-in-an-operatorsource-fail-to-show-up-in-the-ui for reference

Reply:
That is weird. Seems to be working in VS code preview. Let me see what's going on. Thanks for noticing it @galletti94

Reply:
@galletti94 fixed in the latest push. Please review.